src/Renderer/QueryRenderer.ts

```diff
@@ -419,10 +418,11 @@ class QueryRenderChild extends MarkdownRenderChild {
         const amount = 1;
         const timeUnit = 'day';
         const buttonTooltipText = postponeButtonTitle(task, amount, timeUnit);
-        const button = listItem.createEl('a', {
-            cls: 'tasks-postpone' + (shortMode ? ' tasks-postpone-short-mode' : ''),
-            title: buttonTooltipText,
-        });
+        const classes = ('tasks-postpone' + (shortMode ? ' tasks-postpone-short-mode' : '')).split(' ');
+
+        const button = createAndAppendElement('a', listItem);
+        button.addClasses(classes);
```

Comment:
The line 421 is not easily readable IMHO, so I would replace line 424 with something like

```ts
button.addClasses('tasks-postpone');
if (shortMode) {
    button.addClasses('tasks-postpone-short-mode');
}
```

and remove line 421 =) But it is not a big deal, so up to you, so I'm adding this as comment, not a review =)

Reply:
I agree. That is actually how I wrote it first time round, but then went for a smaller diff.

I will change it. Thanks.

Reply:
Oh no. I didn’t have something that simple. Yours is way nicer. Thanks!

Reply:
> The line 421 is not easily readable IMHO, so I would replace line 424 with something like
> 
> ```ts
> button.addClasses('tasks-postpone');
> if (shortMode) {
>     button.addClasses('tasks-postpone-short-mode');
> }
> ```
> 
> and remove line 421 =) But it is not a big deal, so up to you, so I'm adding this as comment, not a review =)

Thanks ... That worked fine. I just needed to use `addClass()` instead of `addClasses()`...